src/Spryker/Zed/DataImport/Business/Model/DataImporter.php

```diff
@@ -123,6 +123,8 @@ public function import(DataImporterConfigurationTransfer $dataImporterConfigurat
 
         $this->beforeImport();
 
+        $startTime = microtime(true);
```

Comment:
I didn't look into tests .. 

BTW I guess I will refactor this to use https://symfony.com/doc/current/components/stopwatch.html